### ISSUE TITLE
The build.sh is not compatible with Inkscape >= 0.92

### ISSUE THREAD

**Issue by wangxa** [keeferrourke/capitaine-cursors#40]
From version 0.92, `inkscape` changes its API, see [this](https://wiki.inkscape.org/wiki/index.php?title=Using_the_Command_Line#Changes_from_0.92). In your `build.sh` script, the option `-e` is never supported, you have to replace it with:
```bash
inkscape -z --export-type=png --export-file="$1/$2/$3/$(basename ${0%.svg}).png" -w $4 -h $4 -d $5 $0
```

**Comment by keeferrourke** [comment]
Hey @wangxa thanks for catching this! I'll get a fix up tonight!

**Comment by keeferrourke** [comment]
Closed by #44
